src/mermaid-isomorphic.ts

```diff
@@ -3,13 +3,16 @@ import { type BrowserType, chromium, type LaunchOptions, type Page } from 'playw
 
 declare const mermaid: Mermaid
 
-const html = import.meta.resolve('../index.html')
+const resolve: (path: string) => string = 
+  import.meta.resolve || ((path) => new URL(path, import.meta.url).href);
+
+const html = resolve('../index.html')
```

Comment:
This is fine, but it could also be written as:

```suggestion
const html = String(new URL('../index.html'))
```